bin/kind-load

```diff
@@ -58,12 +58,13 @@ cluster=${1:-"kind"}
 
 bindir=$( cd "${0%/*}" && pwd )
 
-# shellcheck source=_tag.sh
-. "$bindir"/_tag.sh
-# shellcheck source=_docker.sh
-. "$bindir"/_docker.sh
-
-TAG=${TAG:-$(head_root_tag)}
+if [ -z "$images" ]; then
+  # shellcheck source=_tag.sh
+  . "$bindir"/_tag.sh
+  # shellcheck source=_docker.sh
+  . "$bindir"/_docker.sh
+  TAG=${TAG:-$(head_root_tag)}
+fi
```

Comment:
I was thinking about this a little more and it makes sense CI is failing. We do still need `$TAG` here.

If `--images-host` is passed, we rely on `docker save` being able to save the image tags (`$TAG`) from the remote machine; so we definitely need that.

So, we need `$TAG` if `--images` isn't passed (loading the local working tree), or if `--images-host host` *is* passed (loading the remote working tree)

I think we'll want this to be:

```
if [ -z "$images" ] || [ -n "$images_host" ]; then
  # shellcheck source=_tag.sh
  . "$bindir"/_tag.sh
  # shellcheck source=_docker.sh
  . "$bindir"/_docker.sh
  TAG=${TAG:-$(head_root_tag)}
fi